File: src/tool/spring-boot-generator/output/railway/db/damageOrder.js
let damageOrder = {
	name: "damageOrder",
	description: "伤损单",
	columns: {
		createTime: {
			name: "createTime",
			description: "创建时间",
			nullable: true,
			type: {
				name: "datetime"
			}
		},
		isDelete: {
			name: "isDelete",
			description: "删除标记",
			nullable: false,
			type: {
				name: "int",
				length: null
			}
		},
		orderNo: {
			name: "orderNo",
			description: "伤损单号",
			isPk: true,
			nullable: false,
			type: {
				name: "varchar",
				length: null
			}
		},
		submitStaff: {
			name: "submitStaff",
			description: "提交人",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		},
		submitTime: {
			name: "submitTime",
			description: "提交时间",
			nullable: true,
			type: {
				name: "datetime"
			}
		},
		updateTime: {
			name: "updateTime",
			description: "更新时间",
			nullable: true,
			type: {
				name: "datetime"
			}
		},
		workareaId: {
			name: "workareaId",
			description: "探伤工区",
			nullable: true,
			type: {
				name: "int",
				length: null
			}
		}
	}
};

let columnsArray = [
	"createTime",
	"isDelete",
	"orderNo",
	"submitStaff",
	"submitTime",
	"updateTime",
	"workareaId"
];

function create() {
  return {};
}

damageOrder.columnsArray=columnsArray;
damageOrder.create=create;
module.exports=damageOrder;

File: src/tool/spring-boot-generator/output/railway/railway/src/main/java/com/railway/pojo/entity/DamageOrder.java
package com.railway.pojo.entity;

import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import lombok.Getter;
import lombok.Setter;

import java.util.Date;

@ApiModel("伤损单")
@Getter
@Setter
public class DamageOrder  {

    @ApiModelProperty("创建时间")
    private Date createTime;

    @ApiModelProperty("删除标记")
    private Integer isDelete;

    @ApiModelProperty("伤损单号")
    private String orderNo;

    @ApiModelProperty("提交人")
    private Integer submitStaff;

    @ApiModelProperty("提交时间")
    private Date submitTime;

    @ApiModelProperty("更新时间")
    private Date updateTime;

    @ApiModelProperty("探伤工区")
    private Integer workareaId;

}
